jck/runtime.api/playlist.xml

```diff
@@ -24,6 +24,7 @@
 	-java-args-setup=$(Q)$(EXTRA_OPTIONS)$(Q) \
 	-test-root=$(Q)$(TEST_RESROOT)$(D)..$(D)..$(D)systemtest$(D)stf;$(TEST_RESROOT)$(D)..$(D)..$(D)systemtest$(D)openjdk-systemtest$(Q) \
 	-systemtest-prereqs=$(Q)$(JCK_ROOT)$(Q)  \
+	-java-args-setup=$(Q)$(JVM_OPTIONS)$(Q) \
```

Comment:
there is another -java-args-setup for this test target. If we want to add JVM_OPTIONS, the other should be deleted.

This test target is a custom target but not a production test target, and user defines the options by using EXTRA_OPTIONS. I don't think this test target need to add this line. Your opinion @llxia ?

Reply:
There is no `-java-args=` using in JCK yet, please also update your comment since it is not replacing, it is adding.

Reply:
correct myself, the EXTRA_OPTIONS will be added to JVM_OPTIONS, so keep JVM_OPTIONS is fine. please delete the line of `-java-args-setup=$(Q)$(EXTRA_OPTIONS)$(Q) \`

Reply:
If it's a custom target in which EXTRA_OPTIONS is used to pass in custom jvm options, than adding the JVM_OPTIONS seems redundant and could lead to confusion. I am in favor of not having JVM_OPTIONS for this particular test target.

Reply:
OK. Done.